inductiva/tasks/task.py

```diff
@@ -191,6 +191,26 @@ def _format_data_metric(
     def __repr__(self) -> str:
         return str(self)
 
+    def get_task_time(self) -> str:
+        """Get the task time."""
+        #has not started yet
+        if self.create_time is None:
+            task_time = "Task not started yet (N/A)"
+        #still running
+        elif self.end_time is None:
+            now = datetime.datetime.now(datetime.timezone.utc)
+            create_time = datetime.datetime.fromisoformat(self.create_time)
+            task_time = now - create_time
+            task_time = format_utils.timedelta_formatter(task_time)
+            task_time += " (Task still running)"
+        #task has ended
+        else:
+            end_time = datetime.datetime.fromisoformat(self.end_time)
+            create_time = datetime.datetime.fromisoformat(self.create_time)
+            task_time = end_time - create_time
```

Comment:
When the task ends the metric should be used instead of calculating the duration.